plugins/wasted-bank-space

```diff
@@ -1,2 +1,2 @@
 repository=https://github.com/mcgeer/WastedBankSpace.git
-commit=ccc5614f8e6c4037336fec1150e2c6e1df2cb049
+commit=444819e152dd62f72f6e3ad3186d2296c33caf38
```

Comment:
this one isn't as simple as the previous PRs, but i'm jumping back to 444819e... 1 commit before cat_h https://github.com/mcgeer/WastedBankSpace/compare/4ed1ca904d28c40c52fbcbba7f8900af52c49c75...mcgeer:ccc5614f8e6c4037336fec1150e2c6e1df2cb049